### PR TITLE
RHDEVDOCS-3623: Update Pipeline section in add flow and pipeline docs

### PR DESCRIPTION
**Pull request by HarshCasper**
- JIRA ID: [RHDEVDOCS-3623](https://issues.redhat.com/browse/RHDEVDOCS-3623)
- Aligned team: Pipelines
- SME review: 
- Peer Review:  
- Docs Preview: https://deploy-preview-42868--osdocs.netlify.app/openshift-enterprise/latest/cicd/pipelines/working-with-pipelines-using-the-developer-perspective.html

This PR is for the 4.10 release.

### CODE REVIEW

**Review by vdemeester — APPROVED** [review]

File: cicd/pipelines/working-with-pipelines-using-the-developer-perspective.adoc [hunk 1]
```diff
@@ -28,9 +28,11 @@ After you create the pipelines for your application, you can view and visually i
...
 
-== Creating applications with OpenShift Pipelines
+== Creating OpenShift Pipelines along with applications
 
-To create pipelines along with applications, use the *From Git* option in the *Add* view of the *Developer* perspective. For more information, see xref:../../applications/creating_applications/odc-creating-applications-using-developer-perspective.adoc#odc-importing-codebase-from-git-to-create-application_odc-creating-applications-using-developer-perspective[Creating applications using the Developer perspective].
+To create pipelines along with applications, use the *From Git* option in the *Add+* view of the *Developer* perspective. You can view all of your available pipelines and select the pipelines you want to use to create applications, while importing your code or deploying an image. You will also find a default pipeline selected and can click *Show Pipeline Visualization* to see the pipeline for the application. If an administrator has opted out of Tekton Hub integration, you will be able to see tasks from Tekton Hub which are supported by the cluster. You can also check if a webhook URL exists for a generated pipeline.
```
Comment (Srivaralakshmi) [inline cicd/pipelines/working-with-pipelines-using-the-developer-perspective.adoc:33]: Question: On 2nd thought, I am not sure about the order of the events here that I get to see. What is the 1st thing that I get to see? This is my understanding: A default pipeline that is selected for the application? I understand that despite this default pipeline, I as a user am able to view all the available pipelines and select the one I want to create applications. Correct? 

If so, please rewrite it to suit this order. Something like this:

```suggestion
To create pipelines along with applications, use the *From Git* option in the *Add+* view of the *Developer* perspective. You see a default pipeline selected and can click *Show Pipeline Visualization* to see the pipeline for the application. You can view all of your available pipelines and select the pipelines you want to use to create applications while importing your code or deploying an image. If an administrator has opted out of Tekton Hub integration, you can see tasks from Tekton Hub which are supported by the cluster. You can also check whether a webhook URL exists for a generated pipeline.
```

File: modules/odc-importing-codebase-from-git-to-create-application.adoc [hunk 2]
```diff
@@ -50,7 +50,7 @@ The resource name must be unique in a namespace. Modify the resource name if you
 The *Serverless Deployment* option is displayed in the *Import from git* form only if the {ServerlessOperatorName} is installed in your cluster. For further details, refer to the {ServerlessProductName} documentation.
 ====
 
-. In the *Pipelines* section, select *Add Pipeline*, and then click *Show Pipeline Visualization* to see the pipeline for the application.
+. In the *Pipelines* section, select *Add Pipeline*, and then click *Show Pipeline Visualization* to see the pipeline for the application. You can also see a default pipeline selected for the application and you can choose the pipeline you want. Additionally, administrators can opt out of Tekton Hub integration and the developers will be able to see cluster tasks and tasks which are in the current namespace. You can check out xref:../../cicd/pipelines/understanding-openshift-pipelines.adoc[Understanding OpenShift Pipelines] to learn more about it.
```
Comment (Srivaralakshmi) [inline modules/odc-importing-codebase-from-git-to-create-application.adoc:53]:
```suggestion
. In the *Pipelines* section, select *Add Pipeline*, and then click *Show Pipeline Visualization* to see the pipeline for the application. You can see a default pipeline selected and choose the pipeline you want from the list of available pipelines for the application. Additionally, if an administrator has opted out of Tekton Hub integration, as a developer, you can see cluster tasks and tasks which are in the current namespace.
```

Suggestion: 

- Maintain consistency in content writing. Follow the same style throughout.
- No internal links in modules. Please refer to the [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-hyperlinks-and-cross-references). If you have to mention it in the modules, add the link under the Additional resources section.

**Review by Srivaralakshmi — CHANGES_REQUESTED** [review]
@HarshCasper Left a few suggestions, PTAL. Thanks!

**Review by Srivaralakshmi — APPROVED** [review]
LGTM

File: cicd/pipelines/working-with-pipelines-using-the-developer-perspective.adoc [hunk 3]
```diff
@@ -28,9 +28,11 @@ After you create the pipelines for your application, you can view and visually i
 
 include::modules/op-constructing-pipelines-using-pipeline-builder.adoc[leveloffset=+1]
 
-== Creating applications with OpenShift Pipelines
+== Creating OpenShift Pipelines along with applications
+
+To create pipelines along with applications, use the *From Git* option in the *Add+* view of the *Developer* perspective. You see a default pipeline selected and can click *Show Pipeline Visualization* to see the pipeline for the application. You can view all of your available pipelines and select the pipelines you want to use to create applications while importing your code or deploying an image. If an administrator has opted in for the Tekton Hub Integration, you can see tasks from Tekton Hub that are supported by your cluster. You can also check whether a webhook URL exists for a generated pipeline.
```
Comment (Preeticp) [inline cicd/pipelines/working-with-pipelines-using-the-developer-perspective.adoc:33]: @karthikjeeyar can you please confirm if the last three sentences are technically correct?

[the remark this answers is not in the document]
Comment (karthikjeeyar) [reply cicd/pipelines/working-with-pipelines-using-the-developer-perspective.adoc:33]: We could also add a point about default webhooks that are created when the user goes through the add flow.
eg:
Default webhooks are added for the pipelines that are created using Add flow and URL is visible in the side panel of the selected resources in the Topology view.

**Pull request merged**